tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/AbstractPitclipseSWTBotTest.java

```diff
@@ -134,6 +136,31 @@ public void run() {
         });
     }
 
+    public static void openViewById(String viewId) throws InterruptedException {
+        Display.getDefault().syncExec(()-> {
+            IWorkbench workbench = PlatformUI.getWorkbench();
+            try {
+                workbench.getActiveWorkbenchWindow().getActivePage().showView(viewId);
+            } catch (WorkbenchException e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    public static void closeViewById(String viewId) throws InterruptedException {
+        Display.getDefault().syncExec(() -> {
```

Comment:
similar to what I said above: while opening a view programmatically is good, closing it could be easily done with SWTBot by iterating over the current views...